module_9/task3.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$numbers = [];
+
+for ($i = 0; $i < rand(3, 20); $i++) {
+    array_unshift($numbers, rand(0, 10));
+}
+
+$sumOfNumbers = 0;
+foreach ($numbers as $key => $value) {
+    if ($key % 2 != 0) {
```

Comment:
`($key % 2)` and `($key % 2 != 0)` is the same

Reply:
ohhh exactly, 0 return false, 1-9 return true